src/Storage/Commands.Storage/Blob/Cmdlet/GetAzureStorageBlob.cs

```diff
@@ -114,6 +114,9 @@ public string Container
         [Parameter(Mandatory = false, HelpMessage = "Continuation Token.")]
         public BlobContinuationToken ContinuationToken { get; set; }
 
+        [Parameter(Mandatory =false, HelpMessage ="Run this task in the background and return a job object for tracking execution progress.")]
+        public SwitchParameter AsJob { get; set; }
```

Comment:
@markcowl please regenerate the markdown help file for this cmdlet to add `AsJob`